doc/user/content/sql/patterns/ttl.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Time-To-Live"
+description: "Use the time-to-live pattern and assign an expiration time for a particular row."
+menu:
+  main:
+    parent: 'sql-patterns'
+---
+
+The **time to live (TTL)** pattern helps to remove rows in views, queries, tails, or sinks using expiration times, keeping rows until they are no longer useful, and enabling new use cases.
+
+The following examples are possible TTL use cases:
+
+- Trigger schedule tasks, like emails or messages
+- Store temporal offers for an e-commerce
+- Maintain time-sensitive security blockers
+- Saving-up memory
+
+Before continuing, make sure to understand how [temporal filters](/guides/temporal-filters/) work.
+
+## Pattern
+
+The pattern uses a temporal filter over a row's creation timestamp plus a TTL. The sum represents the row's **expiring time**. After reaching the expiring time, the query will drop the row.
+
+Pattern example:
+```sql
+  CREATE VIEW TTL_VIEW
+  SELECT (created_ts + ttl) as expiring_time
+  FROM events
+  WHERE mz_logical_timestamp() < (created_ts + ttl);
+```
+
+To know the remaining time to live:
+
+```sql
+  SELECT (expiring_time - mz_logical_timestamp()) AS remaining_ttl
+  FROM TTL_VIEW;
+```
+
+## Example
+
+To have a crystal clear understanding of the pattern, let's build a task tracking system. It will consist on a view filtering rows from a table. Each row in the table contains the name, creation timestamp, and TTL of a task.
```

Comment:
```suggestion
For a real-world example of the pattern, let's build a task tracking system. It will consist on a view filtering rows from a table. Each row in the table contains a name, creation timestamp, and TTL of a task.
```